packages/create-vite/index.js

```diff
@@ -41,22 +44,36 @@ const FRAMEWORKS = [
   },
   {
     name: 'vue',
+    display: 'Vue',
     color: green,
     variants: [
       {
         name: 'vue',
-        display: 'JavaScript',
+        display: 'Basic JavaScript',
```

Comment:
Out of curiosity, what's the motivation for displaying "Basic" here? Is there going to be another variant that's not "basic" (e.g., "Advanced JavaScript")? And what would that look like?

(same question for TypeScript below)

Reply:
I also have a slight preference to omit the "Basic" part. I think it expresses the same meaning even with the new `create-vue` and `Nuxt` options

Reply:
I suggested this, but seeing it now implemented, Im good with removing the `basic` 👍